brands/shop/car_repair.json

```diff
@@ -47,6 +47,16 @@
       "shop": "car_repair"
     }
   },
+  "shop/car_repair|Car-X": {
+    "countryCodes": ["us"],
+    "matchNames": ["carx"],
```

Comment:
fyi I'm going to remove this `matchNames` because these aren't needed anymore for variations in case, spacing, punctuation.